extensions/applicationinsights-analytics-js/src/JavaScriptSDK/AnalyticsPlugin.ts

```diff
@@ -512,205 +538,111 @@ export class ApplicationInsights extends BaseTelemetryPlugin implements IAppInsi
                 }
 
                 if (isNullOrUndefined(core)) {
-                    throw Error("Error initializing");
+                    throwError("Error initializing");
                 }
 
                 _base.initialize(config, core, extensions, pluginChain);
-                _self.setInitialized(false); // resetting the initialized state, just in case the following fails
-                let ctx = _self._getTelCtx();
-                let identifier = _self.identifier;
-
-                _self.config = ctx.getExtCfg<IConfig>(identifier);
-
-                // load default values if specified
-                const defaults: IConfig = ApplicationInsights.getDefaultConfig(config);
-                if (defaults !== undefined) {
-                    objForEachKey(defaults, (field, value) => {
-                        // for each unspecified field, set the default value
-                        _self.config[field] = ctx.getConfig(identifier, field, value);
-                        if (_self.config[field] === undefined) {
-                            _self.config[field] = value;
+                try {
```

Comment:
Lots of refactoring to extract out the code that "initializes" the config as this will be reused when the update configuration changes are added.